native/src/main/java/io/ballerina/stdlib/persist/Constants.java

```diff
@@ -33,11 +33,13 @@ private Constants() {
 
     public static final BString PERSIST_CLIENTS = fromString("persistClients");
     public static final String PERSIST_SQL_STREAM = "PersistSQLStream";
+    public static final String PERSIST_GOOGLE_SHEETS_STREAM = "PersistGoogleSheetsStream";
     public static final String PERSIST_IN_MEMORY_STREAM = "PersistInMemoryStream";
     public static final BString KEY_FIELDS = fromString("keyFields");
     public static final String ERROR = "Error";
 
     public static final String RUN_READ_QUERY_METHOD = "runReadQuery";
+    public static final String RUN_READ_AS_TABLE_QUERY_METHOD = "readTableAsStream";
```

Comment:
```suggestion
    public static final String READ_TABLE_AS_STREAM_METHOD = "readTableAsStream";
```

Reply:
updated in [4346c39](https://github.com/ballerina-platform/module-ballerina-persist/pull/201/commits/4346c3959a465aeb05de0872b94ea1056a715a9a)